chatterbot/chatterbot.py

```diff
@@ -234,25 +234,24 @@ def learn_response(self, statement, previous_statement=None):
         """
         Learn that the statement provided is a valid response.
         """
+        from chatterbot.conversation import Statement
+
         if not previous_statement:
             previous_statement = statement.in_response_to
 
         if not previous_statement:
             previous_statement = self.get_latest_response(statement.conversation)
-            if previous_statement:
-                previous_statement = previous_statement.text
 
         previous_statement_text = previous_statement
 
-        if not isinstance(previous_statement, (str, type(None), )):
+        if not isinstance(previous_statement, (Statement, str, type(None), )):
             statement.in_response_to = previous_statement.text
         elif isinstance(previous_statement, str):
             statement.in_response_to = previous_statement
-
-        self.logger.info('Adding "{}" as a response to "{}"'.format(
```

Comment:
Are there any benefits to removing this logging statement?

Reply:
Oh I removed that by mistake during copy paste. My bad.